storage/2020-08-04/queue/messages/peek.go

```diff
@@ -2,94 +2,82 @@ package messages
 
 import (
 	"context"
+	"fmt"
 	"net/http"
+	"strconv"
 	"strings"
 
-	"github.com/Azure/go-autorest/autorest"
-	"github.com/Azure/go-autorest/autorest/azure"
-	"github.com/Azure/go-autorest/autorest/validation"
-	"github.com/tombuildsstuff/giovanni/storage/internal/endpoints"
+	"github.com/hashicorp/go-azure-sdk/sdk/client"
+	"github.com/hashicorp/go-azure-sdk/sdk/odata"
 )
 
+type PeekInput struct {
+	NumberOfMessages int
```

Comment:
```suggestion
	// NumberOfMessages specifies the (maximum) number of messages that should be peak'd from the front of the queue.
	// This can be a maximum of 32.
	NumberOfMessages int
```